tracer/build/_build/Build.Steps.cs

```diff
@@ -1321,24 +1267,18 @@ string GetTestFilter()
             var azureFunctions = TracerDirectory.GlobFiles("test/test-applications/azure-functions/**/*.csproj");
 
             var projects = azureFunctions
-                .Select(x => Solution.GetProject(x))
-                .Where(project =>
-                 (project, project.TryGetTargetFrameworks(), project.RequiresDockerDependency()) switch
-                 {
-                     var (_, targets, _) when targets is not null => targets.Contains(Framework),
-                     _ => true,
-                 });
+                .Where(path =>
+                {
+                    var project = Solution.GetProject(path);
+                    return (project, project.TryGetTargetFrameworks(), project.RequiresDockerDependency()) switch
+                    {
+                        var (_, targets, _) when targets is not null => targets.Contains(Framework),
```

Comment:
Missed your chance to tidy up here 😄 

```suggestion
                        (_, { } targets, _) => targets.Contains(Framework),
```

Reply:
You are right. Fixed here0090d44d568df206e95a77569f063d95511ef735  with another one 😅